extensions/objects/templates/set-of-algebraic-identifier-editor.component.spec.ts

```diff
@@ -19,6 +19,8 @@
 import { GuppyInitializationService } from
   'services/guppy-initialization.service.ts';
 
+// TODO(#11014): Add more extensive front end tests for object editors that rely
+// * on schema editors.
```

Comment:
@shavavo why do you have an extra "*" here?

Reply:
Fixed - Done